plasmapy/physics/parameters.py

```diff
@@ -323,46 +323,46 @@ def ion_sound_speed(*ignore, T_e=0*units.K, T_i=0*units.K,
 @check_quantity({
     'T': {'units': units.K, 'can_be_negative': False}
 })
-def thermal_speed(T, ion="e"):
-    r"""Returns the most probable speed for an ion within a Maxwellian
-    distribution.
+def thermal_speed(T, particle="e"):
+    r"""Returns the most probable speed for an particle within a Maxwellian
+    distributparticle.
```

Comment:
What's a distributparticle? ;)

Reply:
Is a distribution that also accepts electrons.

Reply:
Is a distribution that also accepts electrons.

Reply:
Ooooh wait, I get it. It's like a distribution, but particle? Does this exhaust the pun limit for `physics`? :smile:

Reply:
This is an accidental pun, it's what happens when you try to replace the mentions to `ion` with `particle`. I will fix it :smile: